lib/KCard/index.vue

```diff
@@ -372,6 +409,9 @@
     margin: auto $spacer $spacer;
     overflow: hidden; /* (1) */
   }
+  .footer--empty {
+    height: 40px; /* (1) */
```

Comment:
**[required] Would you remove this class?** We'd like to keep footers heights consistent in a grid so when preserving them it needs to stay at `58px`. This would override it to `40px` for some cards.

The footer already has the height set in the `.footer` class which is applied even when the slot is empty as long as we preserve it, so all should work well without `.footer--empty`.